include/world_builder/world.h

```diff
@@ -71,8 +78,17 @@ namespace WorldBuilder
 
       /**
        * read in the world builder file
-       */
-      void parse_entries(Parameters &prm);
+       * \param prm The Parameters structure to be parsed.
+       * \param max_model_depth This is the maximum model depth. For Cartesian models this
+       * should normally be infinity. For spherical models this should be the radius. For
```

Comment:
As I mentioned in #331, I think it would be useful if WorldBuilder had a complete view of the model. For the sphere, that must include the radius (=max depth). For Cartesian models, you can either define them in a half-space where `x` and `y` are unbounded and you only consider the depth direction special by virtue of saying that `z=0` is special, or you specify in the input not just the x- and y-extents, but also the max depth. For Cartesian models, are you currently specifying the x- and y-extents in the input file?